source/Xunit.Gherkin.Quick.ProjectConsumer/Xunit.Gherkin.Quick.ProjectConsumer.csproj

```diff
@@ -22,10 +22,14 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="Addition\AddTwoNumbers.feature">
+    <None Update="Addition\AddNumbers.feature">
       <Generator></Generator>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```

Comment:
What is this for?

Reply:
Visual studio added that - probably should have skipped that in the commit.